epoxy-processortest/src/test/java/com/airbnb/epoxy/ModelProcessorTest.java

```diff
@@ -130,7 +130,7 @@ public void testModelWithAbstractClass() {
     // We don't generate subclasses if the model is abstract unless it has a class annotation.
     boolean modelNotGenerated;
     try {
-      JavaFileObjects.forResource("ModelWithAbstractClass_.java");
+      JavaFileObjects.forResource("non-sense? Any wrong path will generate IAE");
       modelNotGenerated = false;
     } catch (IllegalArgumentException e) {
       modelNotGenerated = true;
```

Comment:
I suppose this unit-test was broken. There is no ModelWithAbstractClass_.java in the repository, and since I'm not sure of the expected behaviour... can you help on this @elihart ?

Reply:
not sure what this was doing, but it looks like there is a `ModelWithAbstractClass.java` file in test resources that maybe it was supposed to be

Reply:
Actually the tests is in 2 steps, the 1st one assert the ModelWithAbstractClass  (just a compileWithoutErrors), and the 2nd part (highlighted here) was about "not generating subclasses (?)" but I don't understand the case.

If I put the same file on this test, the path is good so it doesn't get IllegalArgumentException and test fails...

Looks like it's a 3 years-old code (you've written in feb. 2017), so maybe some library updates changed the expected behaviour, I don't know how to proceed on this... maybe just delete this 2nd part of the test?

Reply:
Sorry I didn't remove the 2nd part of this test, was waiting your confirmation but it was merged before, oups 😄